book/website/reproducible-research/rdm/rdm-dmp.md

```diff
@@ -0,0 +1,74 @@
+(rr-rdm-dmp)=
+# Data Management Plan
+
+A Data Management Plan (DMP), or Output Management Plan, is a document that describes how your research outputs will be generated, stored, used and shared within your project. 
+A DMP is a living document, which can be updated throughout the research project as needed. 
+
+A Data Management Plan is a roadmap for you to manage your data efficiently and securely. 
+This can prevent data loss or breaches. 
+Planning ahead on how to manage your data consistently can save you time later on! It can also make it easier to {ref}`share<rr-rdm-sharing>` your data with others and therefore make the data more {ref}`FAIR<rr-rdm-fair>`
+
+## A Data Management Plan should provide information on five main topics:
+
+1. **Roles and Responsibilities**
+    * It is important to discuss who is responsible for what during the research project. 
+Defining who is responsible for the management of the data and code can prevent confusion/miscommunication later in the project.
+    * You should check the requirements of your institute and funder. 
+The library research support team of your institute and the website of your funder are usually good places to find information and help. 
+Some of the funders require you to use their Data Management Plan template. 
+You can check if your funder or institute has a Data Management Plan template using [DMPonline](https://dmponline.dcc.ac.uk/).
+2. **Type and size of data collected and documentation/metadata generated**
+    * You can list the file formats you will be using to collect, process and present your data. 
+If you want to share your outputs later on, open file formats that can be opened and used without a particular license for a software programme are preferred, so you may have to convert your files or start working in these formats early on. 
+    * A distinction can be made between different types of data that can be described in the plan separately: 
+        * Raw/primary data: data collected from the source (always keep a read-only version of raw data so you can come back to it later!)
+        * Processed data: a version of the data that has been modified for analysis or visualisation
+        * Finalised data: data that is ready to be shared in a publication or data repository (see {ref}`Sharing and archiving data section <rr-rdm-sharing>` for more information)
+    * All of these types of data will have to be described to be placed into context by using metadata (see the {ref}`Documentation and metadata section<rr-rdm-metadata>`) and adequate documentation which will allow future you, and anyone in your team, to interpret the data. 
+    * It is helpful to know the approximate size (in the range of MB, GB, TB or PB) of the data in these various stages, because this will affect the storage solutions available for you (see below). 
```

Comment:
```suggestion
    * It is helpful to know the approximate size (in the range of MB, GB, TB or PB) of the data in these various stages because this will affect the storage solutions available for you (discussed in the next point). 
```